.github/workflows/publish.yml

```diff
@@ -0,0 +1,19 @@
+name: publish Node.js Package
+
+on:
+  release:
+    types:
+      - published
+jobs:
+  publish-gpr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 12.22.12
+          registry-url: https://npm.pkg.github.com/
+          scope: '@statpro'
+      - run: yarn publish --new-version `echo "${GITHUB_REF##refs/tags/}" | sed -e 's/\./-/3'`
```

Comment:
For reference, if the devextreme-rails gem version is 22.1.6.0.1, then this will publish the package with version 22.1.6-0.1